From: Iain Lane <laney@debian.org>
Date: Wed, 9 Jun 2021 15:37:37 +0100
Subject: tracker-extract/meson.build: Set RPATH for the extract modules

We need to be able to find libtracker-extract.so from pkglibdir.

Forwarded: not-needed
---
 src/tracker-extract/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/tracker-extract/meson.build b/src/tracker-extract/meson.build
index 88ce49f..00d7884 100644
--- a/src/tracker-extract/meson.build
+++ b/src/tracker-extract/meson.build
@@ -114,7 +114,8 @@ foreach module : modules
     dependencies: [tracker_extract_dep, tracker_miners_common_dep] + dependencies,
     include_directories: [configinc, commoninc],
     install: true,
-    install_dir: tracker_extract_modules_dir)
+    install_dir: tracker_extract_modules_dir,
+    install_rpath: tracker_internal_libs_dir)
 
   hash = run_command('calculate-hash.sh', sources)
 
